src/IO/H5/File.hpp

```diff
@@ -191,8 +191,8 @@ const ObjectType& H5File<Access_t>::get(const std::string& path,
     hid_t group_id = std::get<1>(exists_group_name).id();
     if (not std::get<0>(exists_group_name)) {
       current_object_ = nullptr;
-      CHECK_H5(H5Fclose(file_id_),
```

Comment:
In the future please only run clang format on the lines you edited. You can use `git clang-format -f` to do that before you commit.